src/ripple/app/ledger/Ledger.cpp

```diff
@@ -87,7 +87,7 @@ class Ledger::sles_iter_impl
 
     sles_iter_impl (sles_iter_impl const&) = default;
 
-    sles_iter_impl (SHAMap::const_iterator iter)
+    explicit sles_iter_impl (SHAMap::const_iterator iter)
```

Comment:
I'd keep this one as-is. That seems like a reasonable automatic conversion to allow.